fearless/Common/Services/EraValidatorsService/EraValidatorService.swift

```diff
@@ -33,13 +33,13 @@ final class EraValidatorService {
     let providerFactory: SubstrateDataProviderFactoryProtocol
     private var pendingRequests: [PendingRequest] = []
     let operationManager: OperationManagerProtocol
-    let logger: LoggerProtocol
+    let logger: LoggerProtocol?
 
     init(storageFacade: StorageFacadeProtocol,
          runtimeCodingService: RuntimeCodingServiceProtocol,
          providerFactory: SubstrateDataProviderFactoryProtocol,
          operationManager: OperationManagerProtocol,
-         logger: LoggerProtocol) {
+         logger: LoggerProtocol? = nil) {
```

Comment:
What is the purpose of making logger Optional?

Reply:
To allow not passing it in tests

Reply:
Thanks